sklearn/tests/test_multioutput.py

```diff
@@ -425,14 +425,14 @@ def test_multi_output_classification_partial_fit_sample_weights():
     Xw = [[1, 2, 3], [4, 5, 6], [1.5, 2.5, 3.5]]
     yw = [[3, 2], [2, 3], [3, 2]]
     w = np.asarray([2.0, 1.0, 1.0])
-    sgd_linear_clf = SGDClassifier(random_state=1, max_iter=20)
+    sgd_linear_clf = SGDClassifier(random_state=1, max_iter=20, tol=None)
```

Comment:
So with the updated check using the true objective the default tol of 1e-3 is not sufficient for the model to reach the results it was previously reaching?